tools/jenkins-proxy/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.6
```

Comment:
any reason not to use centos or the fedora minimal if you care about size?

Reply:
We don't really care about the size, I can use centos.